packages/datadog-instrumentations/src/check_require_cache.js

```diff
@@ -2,6 +2,27 @@
 
 /* eslint-disable no-console */
 
+/**
+ * Here we maintain a list of packages that an application
+ * may have installed which could potentially conflict with
+ */
+const potentialConflicts = new Set([
+  '@appsignal/javascript',
+  '@appsignal/nodejs',
+  '@dynatrace/oneagent',
+  '@instana/aws-fargate',
+  '@instana/aws-lambda',
+  '@instana/azure-container-services',
+  '@instana/collector',
+  '@instana/google-cloud-run',
+  '@sentry/node',
+  'appoptics-apm',
+  'atatus-nodejs',
+  'elastic-apm-node',
+  'newrelic',
+  'stackify-node-apm'
```

Comment:
told you were missing `sqreen` btw, not sure it's fully incompatible but there are high chances